app/services/forest_liana/query_stat_getter.rb

```diff
@@ -22,7 +22,7 @@ def perform
       case @params['type']
       when 'Value'
         if result.count
-          result_line = result.first
+          result_line = ForestLiana::AdapterHelper.format_live_query_value_result(result)
```

Comment:
Did you test the MySQL behaviour with Pie and Line Charts?

Reply:
https://trello.com/c/MoBYSfLI/2563-live-query-rails-projects-using-mysql-want-to-create-value-pie-line-charts